rnote-compose/src/shapes/arrow.rs

```diff
@@ -0,0 +1,212 @@
+use kurbo::PathEl;
+use na::Rotation2;
+use p2d::bounding_volume::Aabb;
+use serde::{Deserialize, Serialize};
+
+use crate::helpers::{AabbHelpers, Vector2Helpers};
+use crate::shapes::ShapeBehaviour;
+use crate::transform::TransformBehaviour;
+
+use super::Line;
+
+type Radian = f64;
```

Comment:
It's a nice idea, but if we introduce a dedicated radians type we should use it throughout the app, not just here as there are a couple of places where there are angles. In this PR, we can just directly use f64 for the angle. We can always follow up by a refactor later